components/style/values/specified/length.rs

```diff
@@ -458,6 +458,17 @@ impl Parse for Length {
     }
 }
 
+impl Either<Length, None_> {
+    #[inline]
+    #[allow(missing_docs)]
```

Comment:
Please let's try to keep `allow(missing_docs)` to a minimum, can you add a short comment here describing the function instead?

I know other functions in this file have it, but that's just because `deny(missing_docs)` predates them.